shared/features/invoice/tokens/invoice-tokens.ts

```diff
@@ -0,0 +1,65 @@
+export const InvoiceTokens = {
+  header: {
+    sampleTitle: "Your invoice number",
+    receiptTitle: "Receipt NO:",
+    invoiceTitle: "Invoice NO:",
+  },
+  invoiceInfos: {
+    wagmiName: "Wagmi",
+    wagmiAddress: "54 Rue Du Faubourg Montmartre, Paris, France, 75009",
+    wagmiRegistrationNumber: "908 233 638",
+    wagmiEuVATNumber: "FR26908233638",
+    companyName: "Company Name",
+    individualName: "Name",
+    billedTo: "Billed to",
+    siren: "SIREN",
+    vatNumber: "VAT Number",
+    issueDate: "Issue Date",
+    dueDate: "Due Date",
+    destinationAccounts: "Destination Accounts",
+    accountNumber: "Account number",
+    bic: "BIC",
+  },
+  rewardSummary: {
+    title: "Rewards granted on: app.onlydust.com",
```

Comment:
```suggestion
    title: "Rewards granted on: onlydust.com",
```
Seeing as the user can reward on admin.onlydust.com too ?